NN_closure_global_nof_BBL.jl

```diff
@@ -0,0 +1,224 @@
+import Oceananigans.TurbulenceClosures:
+        compute_diffusivities!,
+        DiffusivityFields,
+        viscosity, 
+        diffusivity,
+        diffusive_flux_x,
+        diffusive_flux_y, 
+        diffusive_flux_z,
+        viscous_flux_ux,
+        viscous_flux_uy,
+        viscous_flux_uz,
+        viscous_flux_vx,
+        viscous_flux_vy,
+        viscous_flux_vz,
+        viscous_flux_wx,
+        viscous_flux_wy,
+        viscous_flux_wz
+
+using Oceananigans.BuoyancyModels: ∂x_b, ∂y_b, ∂z_b 
+using Oceananigans.Coriolis
+using Oceananigans.Grids: φnode
+using Oceananigans.Grids: total_size
+using Oceananigans.Utils: KernelParameters
+using Oceananigans: architecture, on_architecture
+using Lux, LuxCUDA
+using JLD2
+using ComponentArrays
+using OffsetArrays
+
+using KernelAbstractions: @index, @kernel, @private
+
+import Oceananigans.TurbulenceClosures: AbstractTurbulenceClosure, ExplicitTimeDiscretization
+
+using Adapt 
+
+include("./feature_scaling.jl")
+
+@inline hack_sind(φ) = sin(φ * π / 180)
+
+@inline fᶜᶜᵃ(i, j, k, grid, coriolis::HydrostaticSphericalCoriolis) = 2 * coriolis.rotation_rate * hack_sind(φnode(i, j, k, grid, Center(), Center(), Center()))
+@inline fᶜᶜᵃ(i, j, k, grid, coriolis::FPlane)    = coriolis.f
+@inline fᶜᶜᵃ(i, j, k, grid, coriolis::BetaPlane) = coriolis.f₀ + coriolis.β * ynode(i, j, k, grid, Center(), Center(), Center())
+
+struct NN{M, P, S}
+    model   :: M
+    ps      :: P
+    st      :: S
+end
+
+@inline (neural_network::NN)(input) = first(neural_network.model(input, neural_network.ps, neural_network.st))
+@inline tosarray(x::AbstractArray) = SArray{Tuple{size(x)...}}(x)
+
+struct NNFluxClosure{A <: NN, S} <: AbstractTurbulenceClosure{ExplicitTimeDiscretization, 3}
+    wT      :: A
+    wS      :: A
+    scaling :: S
+end
+
+Adapt.adapt_structure(to, nn :: NNFluxClosure) = 
+    NNFluxClosure(Adapt.adapt(to, nn.wT), 
+                  Adapt.adapt(to, nn.wS), 
+                  Adapt.adapt(to, nn.scaling))
+
+Adapt.adapt_structure(to, nn :: NN) = 
+    NN(Adapt.adapt(to, nn.model), 
+       Adapt.adapt(to, nn.ps), 
+       Adapt.adapt(to, nn.st))
+
+function NNFluxClosure(arch)
+    dev = ifelse(arch == GPU(), gpu_device(), cpu_device())
+    nn_path = "./NDE_FC_Qb_nof_BBL_trainFC24new_scalingtrain54new_2layer_64_relu_2Pr_model.jld2"
```

Comment:
how big is this file?

Reply:
less that 1MB, maximally perhaps 2MB. Do you think we should host this file in this repository? Or do it somewhere else using something like `DataDeps.jl`